### DISCUSSION TITLE
Could not find com.github.pedroSG94.RootEncoder:library:2.3.2.

### DISCUSSION THREAD

**Discussion by smackia** [pedroSG94/RootEncoder#1364] [Q&A]
![Screenshot 2023-12-12 182358](https://github.com/pedroSG94/RootEncoder/assets/47629915/fda21662-2c79-4880-8062-fa9f5395c71b)

Unable to import the dependency .Please Help

**Comment by smackia** [comment]
@pedroSG94 Please look at this issue i am trying to resolve from yesterday but unable to resolve it

**Comment by pedroSG94** [comment]
Hello,

Are you sure that you added jitpack repository to the project correctly?
Can you share me general build.gradle and app build.gradle?

**Reply by smackia** [reply]
Yes and now i am getting  

Null extracted folder for artifact: ResolvedArtifact(componentIdentifier=com.github.pedroSG94.RootEncoder:rtplibrary:2.2.6, variantName=null, artifactFile=C:\Users\Karan Singh\.gradle\caches\modules-2\files-2.1\com.github.pedroSG94.RootEncoder\rtplibrary\2.2.6\57c507ba6c54a8f218c05a00dca737fb833ce756\rtplibrary-2.2.6.aar, extractedFolder=null, dependencyType=ANDROID, isWrappedModule=false, buildMapping={__current_build__=C:\CodeBase_Bringin\bringindark}, mavenCoordinatesCache=com.android.build.gradle.internal.ide.dependencies.MavenCoordinatesCacheBuildService$Inject@27af2419)

this error

**Reply by pedroSG94** [reply]
Try to add this to gradle.properties:
org.gradle.jvmargs=-Xmx4096M

**Reply by yogeshmehare** [reply]
Hi pedro, even after adding your suggested comment line I am still getting same "Null extracted" issue mentioned above.
Can you please provide a solution. I spent 1 complete day still no luck on overcoming this issue

**Comment by pedroSG94** [comment]
> Hi pedro, even after adding your suggested comment line I am still getting same "Null extracted" issue mentioned above. Can you please provide a solution. I spent 1 complete day still no luck on overcoming this issue

Hello,

Do you have any special reason to use an older version? can you update to last version?

**Reply by yogeshmehare** [reply]
Hi, I am using latest version pedro. In latest version also getting same issue

Null extracted folder for artifact: ResolvedArtifact(componentIdentifier=com.github.pedroSG94.RootEncoder:library:2.4.4, variantName=null, artifactFile=C:\Users\Yogesh M\.gradle\caches\modules-2\files-2.1\com.github.pedroSG94.RootEncoder\library\2.4.4\c6b0a7e1bdebf3eddf72317e8cb1b9004074a3f9\library-2.4.4.aar, extractedFolder=null, dependencyType=ANDROID, isWrappedModule=false, buildMapping={__current_build__=C:\Projects\Android\mobile\OttClient1\OttClient}, mavenCoordinatesCache=com.android.build.gradle.internal.ide.dependencies.MavenCoordinatesCacheBuildService$Inject@3c8ff294)

**Reply by pedroSG94** [reply]
can you share your build files?
Both, the file with the implementation and the file with the maven repo of jitpack

**Reply by yogeshmehare** [reply]
sorry I can't share.

**Reply by pedroSG94** [reply]
This should be an issue in the gradle side. Try to test with this:
https://stackoverflow.com/a/78107676
Also check the following:
- Use the last Android studio version
- Create a new project to make sure that the problem is related with your gradle and the library compile in a fresh project
